apps/cmp.c

```diff
@@ -2328,6 +2409,13 @@ static int save_cert_or_delete(X509 *cert, const char *file, const char *desc)
     }
 }
 
+static int save_crl_or_delete(X509_CRL *crl, const char *file, const char *desc)
+{
+    if (file == NULL)
+        return 1;
```

Comment:
Please use constants instead of MAGICAL NUMBERS.

Reply:
As above